docs/lightning-network/overview.md

```diff
@@ -0,0 +1,23 @@
+# Lightning Network (LN)
+
+---
+
+## What is Lightning (LN)?
+
+The Lightning Network (LN) is a "Layer 2" [protocol](https://en.wikipedia.org/wiki/Lightning_Network) that runs on top of the base layer ("Layer 1") of a blockchain-based cryptocurrency (like Decred). It enables fast, low cost transactions and higher throughput than possible on Layer 1. 
+
+## Decred Lightning Implemetation
+
+Decred's Lightning Network implementation ([dcrlnd](https://github.com/decred/dcrlnd)) is based on the [lnd](https://github.com/lightningnetwork/lnd) implementation for Bitcoin, and is under active development. While the Lightning Network was activated on the Decred mainnet with the activation of [DCP-0004](../../governance/consensus-rule-voting/consensus-vote-archive/#update-sequence-lock-rules) at block [342784](https://dcrdata.decred.org/block/342784) (May 9th, 2019), and is operational on mainnet, it is not recommended to put any significant amounts of DCR into LND nodes until the implementation is more mature. For more on the technical details of LN, see the [dcrlnd](https://github.com/decred/dcrlnd) repo. 
+
+## How Lightning Network works?
```

Comment:
As we discussed, lets remove these two new paragraphs for now and re-introduce them in another PR. This PR is already very large and this can be done as a separate independent change.